languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/multiwords.txt

```diff
@@ -1,4 +1,52 @@
 #French multiwords file used for chunking
+au mieux A
+charge acquéreur N f s
+bas de contention N m sp
+satisfaction client N f s
+relation client N f s
+fiche client N f s
+fiches client N f p
+base de données client N f s
+bases de données client N f p
+un tantinet A
+latina J f sp
+langue maternelle N f s
+mauvais goût N s p
+tout de même A
+Troie Z f s
+a première vue A
+à plusieurs reprises A
+donnée personnelle N f s
+données personnelles N f p
```

Comment:
I don't understand why we need such entries.

Reply:
We are going to be using `multiwords` a lot more to prevent FPs from agreement rules.

Reply:
It is a simple way to disambiguate some phrases, but it should be used only when there is no possible ambiguity. 
Obviously, it is better to disambiguate in a way as general as possible. These entries are now unnecessary because the disambiguation is being done properly for the sentence `J'ai lu la politique de confidentialité des donnés personnelles du site`.

Reply:
When you say 'These entries are now unnecessary because the disambiguation is being done properly for the sentence J'ai lu la politique de confidentialité des donnés personnelles du site.' you mean that the entries for 'donnée(s) personnelle(s) are now unecessary but the other entries are still needed, is that correct?

Reply:
Yes, the other entries are mostly good, with some exceptions. "forte puissance J f s" is not good. We need to disambiguate properly "forte" in disambiguation.xml.